src/common/mfu_daos.c

```diff
@@ -2862,21 +2871,32 @@ int cont_serialize_hdlr(int rank, uint64_t *files_written,  daos_args_t *da,
     daos_handle_t   oh;
     float           version = 0.0;
     herr_t          err = 0;
-    char            filename[150];
-    char            cont_str[130];
-    char            rank_str[8];
+    char            filename[FILENAME_LEN];
+    char            cont_str[FILENAME_LEN];
+    char            rank_str[FILENAME_LEN];
 
     /* init HDF5 args */
     init_hdf5_args(&hdf5);
 
     memset(rank_str, 0, sizeof(rank_str));
     memset(cont_str, 0, sizeof(cont_str));
     memset(filename, 0, sizeof(filename));
```

Comment:
@dsikich Maybe it's okay to use sizeof here since they are static, but I think this is less error prone in the general case:
```suggestion
    memset(rank_str, 0, FILENAME_LEN);
    memset(cont_str, 0, FILENAME_LEN);
    memset(filename, 0, FILENAME_LEN);
```